recipes/densmap-learn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "densmap-learn" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6e9d8a2286335633b924fbb631a0f4fbbb1b6c07714feaae86ec3fc0e9124716
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - numba ==0.48.0
+    - numpy >=1.13
+    - pip
+    - python
+    - scikit-learn >=0.16
+    - scipy >=0.19
+  run:
+    - numba ==0.48.0
+    - numpy >=1.13
+    - python
+    - scikit-learn >=0.16
+    - scipy >=0.19
+
+test:
+  imports:
+    - densmap
+  requires:
```

Comment:
If you are not running any part of the test suite, you can drop the `requires` section here since you don't need `nose` for the simple import test.